libsolidity/interface/StandardCompiler.cpp

```diff
@@ -490,7 +490,12 @@ std::optional<Json::Value> checkOptimizerDetailSteps(Json::Value const& _details
 				);
 			}
 
-			_setting = _details[_name].asString();
+			std::string const fullSequence = _details[_name].asString();
```

Comment:
This file has `using namespace std;`
```suggestion
			string const fullSequence = _details[_name].asString();
```

Reply:
Force of habit